src/screens/Settings/Settings.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { connect } from "react-redux";
+import { bindActionCreators, Dispatch } from "redux";
+// TODO add after update react navigator
+//import { NavigationStackScreenComponent } from 'react-navigation-stack';
+import { ScrollView, KeyboardAvoidingView, Text, View } from "react-native";
+import { translate } from "../../i18n";
+import SettingsEmail from "./SettingsEmail";
+import SettingsPassword from "./SettingsPassword";
+import SettingsPersonalData from "./SettingsPersonalData";
+import SettingsLanguage from "./SettingsLanguage";
+import Loading from "../../components/PaperUIKit/Loading/Loading";
+import ErrorModal from "../../components/PaperUIKit/ErrorModal/ErrorModal";
+import {
+  clearUserError,
+  requestUpdateUserEmail,
+  requestUpdateUserPassword,
+  requestUpdateUserPersonalData
+} from "../../components/User/actions";
+import { UsersStateType } from "../../components/User/types";
+import styles from "./styles";
+
+
+interface SettingStateType {
+  user: UsersStateType
+}
+
+interface SettingsProps {
```

Comment:
`type SettingsProps = ReturnType<typeof mapStateToProps> & ReturnType<typeof mapDispatchToProps>`

just not to repeat all types declarations

Reply:
I think it's easily to read when we describe methods with arguments in props